PlaidLink/Classes/PLDLinkBankMFALoginViewController.m

```diff
@@ -54,17 +67,17 @@ - (void)viewDidAppear:(BOOL)animated {
 - (void)didTapSubmit {
   [_view.submitButton showLoadingState];
   [self.view endEditing:YES];
-  NSDictionary *options = @{
-      @"list" : @(YES)
-  };
+  
+  _options[@"list"] = @(YES);
```

Comment:
Here you can make an NSMutableDictionary and add all the elements from the options variable